deeptime/util/data.py

```diff
@@ -139,12 +149,17 @@ class TimeLaggedDataset:
     ----------
     data : (T, n) ndarray
```

Comment:
Should have same dimension as `data_lagged`, (T, m)?

Reply:
not necessarily